app/assets/stylesheets/application.scss

```diff
@@ -6,3 +6,4 @@
 @import "bento.scss";
 @import "aleph.scss";
 @import "feedback.scss";
+@import "results.scss";
```

Comment:
Since we're really only using this style on results-y things it's easier to know where you're using it if you put it in a sheet named the same as the controller/views where it's used. My preference would be for the styles in bento to be split into different sheets unless they are actually used in multiple places... but that may be worth more discussion. Mostly I find it impossible to ever delete styles if they all get thrown into a shared sheet because you never know if they are used someplace else.